doc/user_guide/large_datasets.rst

```diff
@@ -334,10 +334,34 @@ summary statistics to Altair instead of the full dataset.
 
 VegaFusion
 ~~~~~~~~~~
-If you work with large datasets and require your charts to be interactive,
-make sure to check out the `VegaFusion`_ package. It provides serverside acceleration
-for Altair charts. See its documentation on how to use it as well as the current limitations.
+`VegaFusion`_ is a third-party package that re-implements most Vega-Lite transforms for evaluation
+in the Python kernel.  This makes it possible to scale many Altair charts to millions of rows as long as
+they include some form of aggregation.
+
+VegaFusion 1.0 provides two rendering modes that are useful in different situations.
+
+Mime renderer
+^^^^^^^^^^^^^
+The `VegaFusion mime renderer`_ automates the preaggregate and filter workflow described above. Right before
+a chart is rendered, VegaFusion extracts the datasets and supported transforms and evaluates them in the
+Python kernel. It then removes any unused columns and inlines the transformed data into the chart specification
+for rendering. Charts rendered this way are self-contained and do not require the Python kernel or a custom
+notebook extension to display.
```

Comment:
Maybe add a last sentence hear that charts produced by the mime renderer can be displayed in all the usual frontends where you'd display an Altair chart? Just to drive home that point.